app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
+        *_, in_file, out_file = command.split(" ")
```

Comment:
Don't call command.split() a few times. You should create separate variable for storing result of command.split() and use it where you need